### DISCUSSION TITLE
Github email notification subject changed

### DISCUSSION THREAD

**Discussion by lalo-mx** [apache/netbeans#6511] [General]
Hi,
It seems that the subject line in the notification emails changed, was this intentional?

_Previously:_
[GitHub] [netbeans] mbien commented on pull request #6500: Update jgit from 5.7.0 to 6.7.0.
_Now:_
Re: [I] Run Test Method Action Annotation doesn't show anymore [netbeans]

![image](https://github.com/apache/netbeans/assets/7459881/10b4c655-cdd8-4b72-9933-212dc5027b8e)

**Comment by neilcsmith-net** [comment]
The default subjects were blanket changed across all Apache projects today as far as I know.

**Comment by lalo-mx** [comment]
Is it already reported to Infra?

**Reply by neilcsmith-net** [reply]
They changed it! :wink:

**Reply by neilcsmith-net** [reply]
https://github.com/apache/infrastructure-github-event-notifier/pull/12

**Reply by lalo-mx** [reply]
Thanks, the old way allowed to easily to create a filter and organize my inbox

**Comment by lalo-mx** [comment]
> [apache/infrastructure-github-event-notifier#12](https://github.com/apache/infrastructure-github-event-notifier/pull/12)
